### DISCUSSION TITLE
Retrieve data from previous report

### DISCUSSION THREAD

**Discussion by ndyaanzr** [eclipse-birt/birt#1586] [Q&A]
Hi, I need help with this. Lets say for project A I have Report 1 and 2. Report 1 is created to make an application for changes in project A. While Report 2 is created to apply for other changes in that same Project A, Report 2 also need to display some data from the approved application in previous Report 1. Both reports have their own Report Parameter. Is it possible in birt to display data from Report 1 in Report 2 as well?

**Comment by speckyspooky** [comment]
No, by default you cannot embed 1 report into another report without drillthrough.
But you can add both display options inot one report and via visibility condition you can decide in witch constellation should appear which grid-table constellation.
SO you could create 2 grids, the 1st grid for situation 1 and the 2nd grid for the combination of both and then you can control the display via parameter and the user-selection.

**Comment by hvbtup** [comment]
You can also move most of the logic and layout of Report 1 into a library (e.g. the Data Source, Data Sets, and the layout items). 
Then you can reference these elements from Report 1 as well as from Report 2, so without code redundance, you can "include" Report 1 in Report 2 by referencing the same elements from the library in both reports.
